src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationRepository.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models.Api;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services
+{
+    public class PublicationRepository : IPublicationRepository
```

Comment:
Is there a reason why these methods weren't just added to `GovUk.Education.ExploreEducationStatistics.Admin.Services.PublicationService`?

Reply:
Hey, yeah, it was mainly to make unit testing around permissions an awful lot easier to be honest!  Gives us something to expect to be called using Moq rather than having to set up a lot of standing data.  I might've done it a little differently with hindsight but I'm happy to leave it be like this for now if you're OK with it?

Reply:
Yes that's great, I've approved this PR. Thanks